pipelines/ocw/scripts/pipeline-ocw-www.sh

```diff
@@ -26,19 +26,20 @@ do
   if [[ $branch == "preview" ]]
   then
     OCW_CONTENT_BUCKET=$OCW_CONTENT_BUCKET_PREVIEW
+    VERSION="draft"
   elif [[ $branch == "release" ]]
   then
     OCW_CONTENT_BUCKET=$OCW_CONTENT_BUCKET_RELEASE
+    VERSION="live"
   else
     echo "Invalid branch $1"
     exit 1
   fi
 
   yes | fly -t odl-concourse set-pipeline \
-  -p ocw-www-pipeline-via-vault \
+  -p ocw-www-pipeline-$VERSION \
   --team=ocw \
   --config=pipelines/ocw/pipeline-ocw-www.yml \
-  --instance-var branch=$branch \
```

Comment:
The branch still needs to be set as a var, it just won't be an instance var.

Reply:
What will this var be used for?

Reply:
`((fastly-service-id))-((branch))`?